.github/workflows/ci.yml

```diff
@@ -5,40 +5,42 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-lint-modules-${{ hashFiles('**/yarn.lock') }}
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
-      - run: yarn install
+          node-version: 20
```

Comment:
Why not 20.x?

Reply:
It looks more tidy that way (see the comment below). For 20 it also still installs version 20.6. (and not 20.0.0 or something).

Reply:
I would prefer to have the .x for consistency across all comunica-related workflows. (unless you're willing to change this in _all_ places)